src/AuthModule/components/ChangPass/ChangPass.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useState } from "react";
+import { useForm, SubmitHandler } from "react-hook-form";
+import "react-toastify/dist/ReactToastify.css";
+import styleChangPass from "./ChangPass.module.css";
+import axios from "axios";
+import { useToast } from "../../../context/TostifyContext";
+import { ChangPassProps, FormValues } from "../../../interfaces/Auth";
+
+const ChangPass: React.FC<ChangPassProps> = ({ handleClose }) => {
```

Comment:
The name of the component is  misspelled, it should be "ChangePassword", the "e" is missing from "Change"